lib/OpenQA/Task/Needle/Delete.pm

```diff
@@ -40,6 +41,7 @@ sub _delete_needles ($app, $minion_job, $args) {
         push @{$to_remove{$needle->directory->path}}, $needle;
     }
 
+    $signal_guard->abort(0);
```

Comment:
Shouldn't that be `retry`?

Reply:
I think it would make sense to abort these jobs at any time so users can just attempt the deletion again manually from the UI. Having to do things manually and being confronted with intermediate errors is of course not nice but it is probably better than just a very long waiting time (or even running into a timeout).

It would probably make sense to set `abort => 1` in the beginning and not change anything else later on.

Of course we could also use `retry => 0` but then the job might take too long to complete and will be killed by systemd.

Reply:
For the last comment I've put `abort(0)` once again!